docs/tutorials/notebooks/basic_ot_between_datasets.ipynb

```diff
@@ -0,0 +1,398 @@
+{
```

Comment:
<p>i guess, if i must nitpick, <code>gradients</code> should be <code>gradient</code>,since this is the gradient for the entire point cloud that's shown   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ott-jax/ott/pull/296/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>